src/test/java/reactor/ipc/netty/http/client/HttpClientTest.java

```diff
@@ -245,6 +241,40 @@ public void proxy() throws Exception {
 		page.block(Duration.ofSeconds(30));
 	}
 
+	@Test
+	@Ignore
```

Comment:
ah makes sense... I wonder if we could simulate such a proxy easily? (ie spin a `HttpServer` that serves as the proxy, returning the requested url as a response rather than actually proxying)